optuna/visualization.py

```diff
@@ -29,6 +29,17 @@
     _available = False
 
 
+def is_available():
+    # type: () -> bool
+    """Returns whether visualization is available or not.
+
```

Comment:
IMO, it's kind to leave a description that tells users how to make Optuna's visualization functionality available.
For example:
> :mod:`~optuna.visualization` module depends on ploty version 4.0.0 or higher.
> If your environment hasn't installed the plotly version, this function will return :obj:`False`.
> In such case, please execute  ``$ pip install -U plotly>=4.0.0`` command to install plotly.

Reply:
It's sounds a bit like cramming in the description about the visualization submodule in this function, but I agree that it's better than my current documentation. Let me fix it.

Reply:
Fixed.